asteroid/complex_nn.py

```diff
@@ -176,8 +176,8 @@ def bound_complex_mask(mask: ComplexTensor, bound_type="tanh"):
             "tanh"/"bdt" (default), "sigmoid"/"bdss" or None/"bdt".
 
     References
```

Comment:
Do you know why we have `:` at the end of `Args:` but not for `References`?

Reply:
`Args:` is detected by Sphinx and it handles it well. 
And if I add `:` to `Reference`, it looks like this: 
![image](https://user-images.githubusercontent.com/18496796/100527960-bfc4af80-31d7-11eb-9f07-15c86d06e1bd.png)

I really don't understand the tool, so when I find something that works, I superstitiously stick to it.